### ISSUE TITLE
Add HTTP_PROXY Env Please

### ISSUE THREAD

**Issue by 8Hq7e** [sissbruecker/linkding#362]
Add HTTP_PROXY Env Please,Because I can't directly access some websites

**Comment by sissbruecker** [comment]
Can you please clarify what you are trying to do, what is not working, and what you would expect to happen instead?

**Comment by 0x9394** [comment]
guest OP is in country had internet censorship, and OP has local proxy can bypass it. (and the url OP try to save is censored/blocked by state.)

**Comment by sissbruecker** [comment]
Sorry, I'm still not sure when linkding should make use of a proxy. I'll close this for now, feel free to open a new issue with more details.
